tests/Support/Traits/AssertCss.php

```diff
@@ -29,6 +29,9 @@ private static function getCssNeedleRegExp(string $needle): string
         $needleMatcher = \preg_replace_callback(
             '/\\s*+([{},])\\s*+|(^\\s++)|(>)\\s*+|(?:(?!\\s*+[{},]|^\\s)[^>])++/',
             static function (array $matches): string {
+                /** @psalm-var array{0:string, 1:string, 2:string, 3:string} */
+                $matches = $matches;
+
```

Comment:
Can this instead be `@psalm-param` for the function/closure?

Reply:
see commit message for f6ea5a1